src/VisualStudio/VisualStudioInteractiveComponents/AssemblyRedirects.cs

```diff
@@ -5,17 +5,17 @@
 
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.Scripting.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.CSharp.Scripting.dll")]
-[assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
+// [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
```

Comment:
Interactive doesn't actually ship VB.  All of these binding redirects were working just by chance because VB built into the same directory.